borg/archiver.py

```diff
@@ -1720,6 +1720,8 @@ def run(self, args):
         setup_logging(level=args.log_level, is_serve=args.func == self.do_serve)  # do not use loggers before this!
         if args.show_version:
             logger.info('borgbackup version %s' % __version__)
+        if 'output_list' in args and args.output_list:
```

Comment:
that is a bit strange:
"in" checks for a dict member(?), but then you access an attribute?